src/main/scala/org/broadinstitute/dsde/agora/server/business/AclBusiness.scala

```diff
@@ -0,0 +1,122 @@
+package org.broadinstitute.dsde.agora.server.busines
+
+import com.google.api.services.storage.model.{ObjectAccessControl, BucketAccessControl}
+import org.broadinstitute.dsde.agora.server.business.{EntityAuthorizationException, NamespaceAuthorizationException}
+import org.broadinstitute.dsde.agora.server.dataaccess.acls.AgoraPermissions.Manage
+import org.broadinstitute.dsde.agora.server.dataaccess.acls.{AgoraPermissions, AuthorizationProvider}
+import org.broadinstitute.dsde.agora.server.model.{AgoraEntity, AgoraEntityType}
+import org.broadinstitute.dsde.agora.server.dataaccess.acls.gcs.GcsClient._
+import spray.routing.RequestContext
+
+//TODO: Lots to refactor in this class. too much duplication
+class AclBusiness(authorizationProvider: AuthorizationProvider) {
+
+  def listNamespaceAcls(context: RequestContext, entity: AgoraEntity, user: String) = {
+    if (authorizationProvider.namespaceAuthorization(entity, user).canManage) {
+      val bucket = entityToBucket(entity)
+      val bucketAcls = listBucketAcl(bucket).setFields("items(entity,role)").execute()
+
+      bucketAcls.toPrettyString
+
+    } else {
+      throw new NamespaceAuthorizationException(AgoraPermissions(Manage), entity)
+    }
+  }
+
+  def insertNamespaceAcl(_context: RequestContext, entity: AgoraEntity, user: String, acl: BucketAccessControl) = {
```

Comment:
typically, they are used to avoid naming conflicts (i.e, if there was another context variable in scope). In this case, there is not, so the _ can be removed.